packages/js/product-editor/src/components/custom-fields/custom-field-name-control/style.scss

```diff
@@ -0,0 +1,43 @@
+.woocommerce-custom-field-name-control {
+	background-color: #fff;
+
+	&.has-error {
+		.components-combobox-control__suggestions-container {
+			> .components-flex,
+			> .components-form-token-field__suggestions-list {
+				border-color: $studio-red-50;
+			}
+		}
+	}
+
+	.components-combobox-control__suggestions-container {
+		position: relative;
+		border: none;
+
+		&:focus-within {
+			> .components-flex {
+				border-color: var(--wp-admin-theme-color);
+				box-shadow: 0 0 0 0.5px var(--wp-admin-theme-color);
+			}
+		}
+
+		> .components-flex {
+			height: 36px;
```

Comment:
I think we can use the variable `$gap-largera` here.

Reply:
I thought we were not using those vars anymore instead we should use `$grid-unit` but I did not use it in this case since 36 does not fit any value from the grid system.